### PR TITLE
Update default key generated to SECP256k1

### PR DESCRIPTION
**Pull request by harshadptl**
* **The major changes of this PR**

* **The major impacts of this PR**
  - [ ] Impact WASM?
  - [ ] Impact mainnet data compatibility?

* **Extra documentations**

### CODE REVIEW

File: api/src/xfr/sig.rs [hunk 1]
```diff
@@ -511,6 +511,17 @@ impl XfrKeyPair {
         }
     }
 
+    pub fn generate_secp256k1_from_bytes(bytes: &[u8]) -> Result<Self> {
```
Comment (findora-crypto) [inline api/src/xfr/sig.rs:514]:
```suggestion
    /// Generate a key pair from secret key bytes/
    pub fn generate_secp256k1_from_bytes(bytes: &[u8]) -> Result<Self> {
```

**Comment by findora-crypto** [comment]
@harshadptl can you take a look at the failed tests?

**Comment by harshadptl** [comment]
i'll check

**Comment by confuseSUN** [comment]
The random number generated by chacha for the second time is equal to  [218, 65, 89, 124, 81, 87, 72, 141, 119, 36, 224, 63, 184, 216, 74, 55, 106, 67, 184, 244, 21, 24, 161, 28, 195, 135, 182, 105, 178, 238, 101, 134],  and using the secp256k1 generated by this random number to decrypt the memo will fail .
       ` let ret = [218, 65, 89, 124, 81, 87, 72, 141, 119, 36, 224, 63, 184, 216, 74, 55, 106, 67, 184, 244, 21, 24, 161, 28, 195, 135, 182, 105, 178, 238, 101, 134];
        let xfr_key = XfrKeyPair::generate_secp256k1_from_bytes(&ret).unwrap();
        let (bar, memo) =
        conf_blind_asset_record_from_ledger(&xfr_key.pub_key, 75, ASSET1_TYPE);
    
        //will fail
         memo.decrypt_amount_and_asset_type(&xfr_key).unwrap();`

**Pull request merged**